libethcore/SealEngine.cpp

```diff
@@ -43,9 +43,11 @@ void SealEngineFace::populateFromParent(BlockHeader& _bi, BlockHeader const& _pa
 	_bi.populateFromParent(_parent);
 }
 
-void SealEngineFace::verifyTransaction(ImportRequirements::value _ir, TransactionBase const& _t, BlockHeader const&) const
+void SealEngineFace::verifyTransaction(ImportRequirements::value _ir, TransactionBase const& _t, BlockHeader const& _bi) const
 {
-	if (_ir & ImportRequirements::TransactionSignatures)
+	if (_bi.number() < chainParams().u256Param("metropolisForkBlock") && _t.hasZeroSignature())
```

Comment:
Why don't we check for `_ir &ImportRequirements::TransactionSignatures` here?
Also: Is the zero signature considered invalid already now because of the v value?